google/ggrc-core
# GGRC-6042 Renamed Task group title to Title on Setup tab and Edit Task popup

# Issue description

Task group title is named Summary on Setup tab and Edit Task popup.

# Steps to test the changes

- Run build_assets command.
- Run launch_ggrc application.
1. Create workflow
2. Open Setup tab
Actual: Title column is named Summary
Expected: Title column should be named Title
3. Click some task group
4. Click Edit Task Group in 3bbs
Actual: Title field is named "Summary" in Edit popup
Expected: Title field is named "Title" in Edit popup

# Solution description

 Renamed Task group title to Title on Setup tab and Edit Task popup

# Sanity checklist




# PR Review checklist
